media/js/products/vpn/thanks.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// Create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+(function (Mozilla) {
+    'use strict';
+
+    var DownloadThanks = {};
+
+    /**
+     * Determine if browser should attempt to download Mozilla VPN on page load.
+     * @param {String} platform
+     * @returns {Boolean}
+     */
+    DownloadThanks.shouldAutoDownload = function (platform) {
+        var supportedPlatforms = ['windows', 'osx'];
+
+        if (supportedPlatforms.indexOf(platform) !== -1) {
+            return true;
+        }
+
+        return false;
+    };
+
+    /**
+     * Get the VPN download link for the appropriate platform.
+     * @param {Object} window.site
+     * @returns {String} download url
+     */
+    DownloadThanks.getDownloadURL = function (site) {
+        var prefix = 'vpn-download-link-';
+        var link;
+        var url;
+
+        switch (site.platform) {
+            case 'windows':
+                link = document.getElementById(prefix + 'win');
+                break;
+            case 'osx':
+                link = document.getElementById(prefix + 'mac');
+                break;
+        }
+
+        if (link && link.href) {
+            url = link.href;
+        }
+
+        return url;
+    };
```

Comment:
Issue (blocking) I'm not sure we need this either? Right now if I'm on mac and click to try and download a windows installer, I get nothing since this is doing a platform based check.

I don't think we need to keep this as copy over from how we do Firefox downloads, since here we have two separate download pages and not a single page that handles all platforms.

We can probably remove this file entirely, and just have a separate JS bundle for both the Windows and Mac download pages, that triggers the correct file download automatically? This feels like it might be easier to maintain than a shared file?

Let me know if you want to talk through this if something I've said doesn't make sense 🙂